assets/app/features/upgrade/controllers/upgrade-backup.controller.spec.js

```diff
@@ -21,15 +21,22 @@ describe('Upgrade Flow - Backup Controller', function() {
         mockedDownloadResponse = {
             data: mockedDownloadFile,
             'headers': function() {}
+        },
+        upgradeVm = {
+            steps: {
+                activeStep: {
+                    finished: false
+                }
+            }
         };
 
     beforeEach(function() {
         //Setup the module and dependencies to be used.
         bard.appModule('crowbarApp');
         bard.inject('$controller', '$rootScope', '$q', '$httpBackend', '$document', 'crowbarBackupFactory');
-
+        $rootScope.upgradeVm = upgradeVm;
```

Comment:
Is this as to create a new scope on each test? Im guessing than using $rootScope directly means Im reusing the same scope on each test which could lead to instability on the tests that use the scope right?